tests/test_batch.py

```diff
@@ -931,3 +934,14 @@ def test_cancel_batches_success(
         )
 
         assert mock_request.last_request.json() == {"batch_ids": batch_ids}
+
+    def test_sequence_builder_field_is_accessible(
+        self,
+        mock_request: Any,
+    ):
+        """Simple test verifying the property sequence_builder
+        of a batch is accessible"""
+        response = self.sdk.get_batches()
+        first_batch = response.results[0]
+        assert first_batch.sequence_builder == self.pulser_sequence
+        assert mock_request.last_request.method == "GET"
```

Comment:
maybe assert that the get batch endpoint is called. I would amend the description to clarify that the get batches endpoint doesn't return the sequence builder, but the sequence builder can be accessed anyway and will trigger a call to the get batch endpoint